### ISSUE TITLE
Reserved weapon stats

### ISSUE THREAD

**Issue by BlackVegetable** [akintu/akintu#22]
I can see a lame situation on the horizon.  

Kyle and I proposed that characters be allowed two sets of weapons to switch between at will.  One could be a bow, and the other a couple of swords dual-wielded.  However...

How should we balance between someone that has two different two handed weapons set up vs. someone with two sets of two weapons set up?  The latter will have twice the possible attributes attached to their weapon!  

A couple of options...

A. Only stats from the currently equipped set matter.  (Causes some issues with max HP/MP fluctuating in battle among other things.)

B. Double stats attributed to two-handed weapons.  (Likely causes issues I cannot currently detect.)

C. Split the stats on weapons into two categories-- weapon specific stats which handle things like on-hit chance to cause bleeding (which should be an advantage of two weapons) and character stats such as + Constitution, which are doubly applied to two-handed weapons.

Any suggestions? @basepi @Jzar

**Comment by BlackVegetable** [comment]
I think I solved this issue.  It is almost just C.  

We don't split the stats on weapons on item creation.  We _do_ double the effect of any "character stat" gains on two-handed weapons though.  I'll specify what that means exactly on a page soon to follow.

If this implementation sounds like a bad idea, please discuss it.  Otherwise, feel free to close this issue.

**Comment by basepi** [comment]
I'm down with either A or C.  A is what I was thinking, but C would be fine too.

**Comment by BlackVegetable** [comment]
C solves more issues than A does.  A also introduces problems when swapping weapons costs no AP.  You could injure yourself just by equipping a weapon with less +Constitution, equipping your other weapon, and then swapping again... (just one example.)

**Comment by basepi** [comment]
Sounds fine.  C it is, unless @Jzar has issues with that.

**Comment by malikoth** [comment]
When generating a 2H weapon's stats, why not just allow twice the cap for character stat budget, and then when the character looks at it, it will be very obvious what its stats are.  Then they won't be surprised when they put it on and find it's twice as good as they thought.  And on the other hand, they won't pass over 2H weapons in favor of dual 1Hs because the one-handers appear to have twice the total stats.

So basically just your C solution, but display the doubled stat on the weapon all the time.

**Comment by BlackVegetable** [comment]
That's what I meant from the start.  Sorry I didn't communicate it properly.
